config.json

```diff
@@ -54,6 +54,21 @@
             "required": false,
             "type": "textarea",
             "repeatable": false
+        },
+        {
+            "key": "use_goprod",
+            "name": "Use goprod module?",
+            "type": "radio",
+            "choices": [
+                {
+                    "name": "No",
+                    "value": ""
```

Comment:
Less interestingly, setting this value to null makes it appear that no choice is set when the user reopens the configuration